postinstall.js

```diff
@@ -1,4 +1,9 @@
 const shell = require('shelljs');
 
-shell.exec('sed -i \'s/PLATFORM_JAVA_OPTS/#PLATFORM_JAVA_OPTS/\' .env');
-shell.exec('docker-compose build');
+/*
+Neither of the below items are currently needed. The `PLATFORM_JAVA_OPTS` should not be commented out and the current
```

Comment:
If they are no longer needed and are commented out should the file be removed?
Also the comment says `PLATFORM_JAVA_OPTS` shouldn't be commented out but it is.
Unclear what the intent is here or if it's local changes that make it up

Reply:
File was left in for reference, but now removed.